docs/event-bus/03-04-knative-eventing-mesh.md

```diff
@@ -41,3 +41,76 @@ To reach the new Eventing Mesh, use an HTTP request with the `/events` path.
 For example, if you have used `gateway.example.cx/v1/events` so far, use `gateway.example.cx/events` to make sure you work with the new Eventing Mesh. 
 
 >**NOTE:** The HTTP source adapter only accepts Events compliant with the [CloudEvents 1.0 specification](https://github.com/cloudevents/spec/blob/v1.0/spec.md).
+
+## Other Channel options
+
+By default Kyma comes with Natss but Knative eventing allows to exchange the default channel implementation. It even allows to have multiple channel implementations simultaneously.
+The default channel implementation can be changed during installation using an installation-override like this:
+
+```bash
+$ cat << EOF | kubectl apply -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: knative-eventing-overrides
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    component: knative-eventing
+    kyma-project.io/installation: ""
+data:
+  knative-eventing.channel.default.apiVersion: knativekafka.kyma-project.io/v1alpha1
+  knative-eventing.channel.default.kind: KafkaChannel
+EOF
+```
+
+In this example the default channel is set to Kafka.
+
+### Kafka
+
+There is a Knative compatible [Kafka channel implementation](https://github.com/kyma-incubator/knative-kafka) which can be used for more production-ready eventing workloads but is still in alpha state.
+
+>**NOTE:** Kafka Channel integration is in alpha version. Use it only for testing purposes.
+
+The Knative channel implementation supports [these providers](https://github.com/kyma-incubator/knative-kafka/blob/9eb3fa3f6e67ffc80b162d2ef4c8a8a3942d9c5f/resources/README.md#kafka-providers):
+
+1. [Azure Event Hubs](https://azure.microsoft.com/en-us/services/event-hubs/)
```

Comment:
```suggestion
* [Azure Event Hubs](https://azure.microsoft.com/en-us/services/event-hubs/)
```
For this one and the following two points, I changed the numbering to bullet points - it's not a procedure, so it's better to use regular bullet points.